src/main/java/seedu/ui/Ui.java

```diff
@@ -534,11 +534,16 @@ public boolean getResetReconfirm(CommandType commandType) {
         printSeparatorLine();
         printUserInputMessage();
         String confirm = readUserInput();
-        if (!confirm.equalsIgnoreCase("y")) {
-            printSeparatorLine();
-            printLine("Shutting down the command...");
+        if (confirm.equalsIgnoreCase("y")) {
+            return true;
+        }
+        printSeparatorLine();
+        String errorMessage = "Shutting down the command...";
```

Comment:
could you initialize errorMessage as a String first, assign it inside the if else calls?
Also, I think it should be message now since inputting "n" message should not be an error.

```
if (!confirm,equalsIgnoreCase("n") {
    msg= [msg]
}
msg += Shutting down...
    
printSep();
printLine(msg);
return false;
```